### PR TITLE
Development

### PR DESCRIPTION
**Pull request by jakedavison**

### CODE REVIEW

File: Wander/app/src/main/java/com/teamwan/wander/Feedback.java [hunk 1]
```diff
@@ -138,50 +146,103 @@ private void setTypefaces() {
...
+            values = data.getOffTaskCorrectnesses();
+            for (float v: values) {
+                avg += v;
+            }
+            return avg/values.size();
+        }
+    }
+
+    public float fetchBarDataAcc(int task){
+        ArrayList<Float> values;
```
Comment (PJUllrich) [inline Wander/app/src/main/java/com/teamwan/wander/Feedback.java]: This functionality is duplicate with the previous function. The only difference is that it uses a different set of values

File: Wander/app/src/main/java/com/teamwan/wander/FeedbackFragmentChart.java [hunk 2]
```diff
@@ -69,11 +79,19 @@ private void initialiseFragment(){
         if (titleString.length()>20) {
             title.setTextSize(26);
         }
-        if (chartData instanceof LineData) {
-            chart.setData((LineData) chartData);
-            chart.invalidate();
+        if (chartTypeLine) {
```
Comment (PJUllrich) [inline Wander/app/src/main/java/com/teamwan/wander/FeedbackFragmentChart.java]: These 2 if statements have the same conditions and can be combined into 1

**Review by PJUllrich — APPROVED** [review]
Just some small comments to improve code quality

File: Wander/app/src/main/java/com/teamwan/wander/FeedbackFragmentChart.java (hunk 2 above)
Comment (ThomasdenH) [reply Wander/app/src/main/java/com/teamwan/wander/FeedbackFragmentChart.java]: I don't know if I follow, but if you mean `if (titleString.length()>20)` and `if (chartTypeLine)` I disagree. The titles might be changed or a chart could be added so which titles are long might change.

Comment (jakedavison) [reply Wander/app/src/main/java/com/teamwan/wander/FeedbackFragmentChart.java]: I believe this is solved, was just a small leftover implementation.

**Pull request merged**